slurm_state/config.py

```diff
@@ -90,6 +90,12 @@ def _valid(value):
     return _valid
 
 
+def alloc_valid(value):
+    if value == "*":
+        return value
+    return string_list(value)
+
+
```

Comment:
This version of the function should probably be removed

Reply:
Thank you! This is done